host/xtest/xtest_10000.c

```diff
@@ -280,7 +280,7 @@ static const uint8_t pbkdf2_3_dkm[] = {
 #define pbkdf2_4_password pbkdf2_1_password
 #define pbkdf2_4_salt pbkdf2_1_salt
 #define pbkdf2_4_iteration_count 16777216
-static const uint8_t pbkdf2_4_dkm[] = {
+static const uint8_t pbkdf2_4_dkm[] __maybe_unused = {
```

Comment:
Ah, it's OK. `gcc -E` shows that the include sequence is `xtest_helpers.h` -> `tee_api_types.h` -> `compiler.h`. So it comes by default in the Linux side of `xtest` thanks to the `host_include` folder of the TA devkit. A "regular" client application accessing only the `libteec` API would not have it however.
As for the TA side, it can include `<compiler.h>` or `<tee_api_types.h>`, that's included in the TA devkit.